processes/bug_bounty_criteria.md

```diff
@@ -0,0 +1,35 @@
+# Bug Bounty Criteria
+
+The following document describes the criteria for eligibility of monetary reward for security researchers
+who engage with the Node.js Security Working group through the Node.js Ecosystem 
+[bug bounty program](https://hackerone.com/nodejs-ecosystem/), as is currently running on HackerOne.
+
+## Module Characteristics
+
+1. **Module download count** - x >= 1000 downloads a month which accounts for 7% of npm packages 
+(courtesy of @ChALkeR here https://github.com/nodejs/security-wg/issues/151#issue-303209104)
+2. **Approved Modules** - A list of modules where their maintainers approved to be included in the
```

Comment:
Because of the line-wrapping, this renders as `1.` rather than `2.`. Indent the line above (and the line below) to achieve the desired effect.

Reply:
Is this placeholder text? I'm not sure I understand how this is a Module Characteristic? Maybe better to replace the two list items with two sentences in a paragraph? Something along these lines?:

```
Only modules that have more than 1000 downloads per month from npm are eligible.
The program only includes modules where the maintainers approve its inclusion.
```

Reply:
Oh, I see. This is a somewhat internal-facing document, not something that is necessarily meant to be broadcast to a wide external audience. You can probably ignore a lot of my comments that I've already left, but I'll leave them anyway as I do believe they do improve the text, if only slightly.

Reply:
I'm not sure that this is internal facing.  It's for those who might want to claim the bounties....

Reply:
@Trott I didn't follow on why the 1. and 2. don't render well...? and yes this is supposed to be a process that is public so generally speaking it is public facing indeed.

I think it's a good idea at the moment to keep the messaging as is and chalker's reference to provide proper context as this is still a work in progress.